### DISCUSSION TITLE
"Objects" section removed in favour of CLIP, can we still get it back?

### DISCUSSION THREAD

**Discussion by iUltimateLP** [immich-app/immich#5295] [Q&A]
Hey, I've noticed that the "Objects" section in Immich is gone now. After researching for a bit, I found this answer by @alextran1502 on Reddit:

> Hello we have removed object detection in places of contextual CLIP search which is far more superior and accurate then traditional object detection search

I do see that CLIP is _much_ more accurate than the previous version, but I miss the list of things I could scroll through. Is there a way to get the list to work _with_ CLIP? Or does such a list-based approach not work with CLIP?

Thanks!

**Comment by alextran1502** [comment]
Hello, CLIP data is a list of number, thus, showing a list of object is not possible
